content/installation/_index.md

```diff
@@ -6,6 +6,12 @@ weight = 10
 toc = true
 +++
 
+{{% notice warning %}}
+The https://infracloudio.github.io/charts Helm repository is deprecated. In the future, BotKube charts from https://infracloudio.github.io/charts can be garbage collected and no longer available.
+The new BotKube Helm repository is available under https://charts.botkube.io.
+{{% /notice%}}
```

Comment:
Do you think we need such notice on the website? 🤔 I would only post it on GitHub release info, as this is the place where existing users would look for changes, but I don't know 🤔 My only concern is that we will need to eventually remove this 😄 We can keep it, but let's set a reminder after 3 months or so 😄

Reply:
It was also discussed with Blair to put it in doc. And definitely that should be removed. IMO, before the 0.14 release. WDYT? So we will have it as you mentioned for ~3months.